Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -33,6 +33,12 @@
             "isExpanded": false,
             "visibleRule": "WebAppKind != applinux && WebAppKind != \"\""
         },
+        {
+            "name": "PostDeploymentAction",
+            "displayName": "Post Deployment Action",
+            "isExpanded": false,
+            "visibleRule": "WebAppKind != \"\""
```

Comment:
What does this indicate? Shouldn't it show up in call cases now

Reply:
This means that the webAppKind should not be empty.